docs/cloud-soar/introduction.md

```diff
@@ -0,0 +1,670 @@
+
+#### Create a custom incident template
+
+Incident templates define which attributes will be automatically set each time an incident is generated. These attributes include incident type, classification, assignment, playbooks, and many others. For example, you may want to automatically assign certain incoming incidents to a certain analyst based on their timezone or area of expertise. Or, you might want to automatically run a playbook on all incoming incidents. For more information, see [Incident templates](/docs/cloud-soar/automation/#incident-templates).
+
+In this section, we’ll create a custom incident template. This template will automatically assign the playbook you created earlier to certain new incidents, and then automatically run it.  
+
+1. [**Classic UI**](/docs/cloud-soar/overview#classic-ui). Click the gear icon <img src={useBaseUrl('img/cloud-soar/cloud-soar-settings-icon.png')} alt="Settings menu icon" style={{border: '1px solid gray'}} width="25"/> in the top right, select **Automation**, and then select **Incident templates** in the left nav bar. <br/>[**New UI**](/docs/cloud-soar/overview#new-ui). In the main Sumo Logic menu select **Automation > Template**. You can also click the **Go To...** menu at the top of the screen and select **Template**.
+1. Near the top, click the **+** icon to create a new template.
+1. In the **Name** provide a name for the template. 
+1. In the **Category** field enter a category, for example, **Test**. 
+1. Click the **Incident** tab.
+1. Leave the fields as their defaults, and select **General** for **Type**.
+1. Click **Apply** to create the template. The template is displayed.
+1. Click the **+** icon next to **Playbook** to add a new playbook.
+1. Select the check mark next to the playbook you created in the previous section. 
+1. Click **Add**.
+1. Toggle the **Autorun** switch to the **Enabled** (blue) position.
+
+#### Create a custom automation rule
+
+Automation rules define what happens when data is received from a source. These rules allow specific data to be parsed from the incoming data sources and then acted upon automatically. For more information, see [Automation rules](/docs/cloud-soar/automation/#automation-rules).
+
+Automation rules can automatically pull information from sources. They can also execute playbooks based on certain criteria. For example, you might want to pull Insights from Cloud SIEM once an hour and create incidents from them. Then, you could configure a rule that runs a playbook based on the tags in the incident. This way, the entire incident response cycle is automated: Cloud SIEM identifies a threat, and playbooks in Cloud SOAR are automatically deployed to contain and eradicate the threats, restore systems, and email a final report for an analyst to review.  
+
+Let's create a custom automation rule. This rule will pull information from Cloud SIEM every five hours.
```

Comment:
Although I don’t think we have an official style guide entry on this, I believe our convention is to write all numbers as numerals rather than spelling them out. (As an AP Stylebook follower, it pains me to say that!).

```suggestion
Let's create a custom automation rule. This rule will pull information from Cloud SIEM every 5 hours.
```